pkg/network/setup/netstat.go

```diff
@@ -165,11 +170,20 @@ func ifacesStatusFromDomainInterfaces(domainSpecIfaces []api.Interface) []v1.Vir
 			Name:       domainSpecIface.Alias.GetName(),
 			MAC:        domainSpecIface.MAC.MAC,
 			InfoSource: netvmispec.InfoSourceDomain,
+			QueueCount: domainInterfaceQueues(domainSpecIface.Driver),
```

Comment:
@oshoval what do you think about adding the default value during the unmarshaling. Something similar to what is done with [alias](https://github.com/kubevirt/kubevirt/blob/5bf9394089f6790279aad9ac2001a7a230d34ec3/pkg/virt-launcher/virtwrap/api/schema.go#L817-L816)?

Reply:
The default logic you described will still be kept.
The unmarshal will put 1 to the `domain.Driver.Queues` in case the json/xml value is empty. The `omitempty` on `VirtualMachineInstanceNetworkInterface.QuequeCount` will be kept.

Reply:
Even libvirt itself removes the 1 at `domain.Driver.Queues` when we set it (for example with multi queue enabled + one vCPU), i don't think we should change it. (not i think it deserves it imo)
and for sure not on this PR even if so please, as it touches `domain.Driver.Queues` not queueCount

Reply:
I like @AlonaKaplan proposal and I would have tried to do that. It looks cleaner, removing logic from the one interpreting what is the default.

At the same time, I think we could let it in as is and attempt this in a following PR.

Reply:
Can be done in a follow-up

Reply:
Thanks